### ISSUE TITLE
Sorry, I want to know the specific algorithm of item similarity recommender

### ISSUE THREAD

**Issue by tjnu1430130310** [apple/turicreate#559]
Hello, I just started learning machine, English is not my mother tongue, I'm very sorry. 

I built a recommendation system using GraphLab Create and Django. Now I really want to know whether the algorithm behind the use of item_similarity_recommender modeling is collaborative filtering. In addition, can you tell me more about whether it is user-based collaborative filtering or item-based collaborative filtering? 

I know I shouldn't ask this question to you. I'm very sorry, but I'm always puzzled and I'm so sorry.

**Comment by tjnu1430130310** [comment]
In here, https://apple.github.io/turicreate/docs/api/generated/turicreate.recommender.item_similarity_recommender.create.html#turicreate.recommender.item_similarity_recommender.create

Create a recommender that uses item-item similarities based on users in common.

Does this sentence mean that it is the user-based collaborative recommendation?

**Comment by boulbi777** [comment]
Hello,
why did you close this issue ? I'm really interested since it's been confusing for me.
Have you found a solution ? If so I'd be glad to have a response !
Thank you !
👍

**Comment by nickjong** [comment]
Yes, this toolkit is an implementation of collaborative filtering. For some more details, try looking at the relevant C++ header: 
https://github.com/apple/turicreate/blob/master/src/unity/toolkits/recsys/models/itemcf.hpp
